log-collector-script/linux/eks-log-collector.sh

```diff
@@ -474,6 +474,8 @@ get_k8s_info() {
 
       cp --force --recursive --dereference /etc/kubernetes/kubelet/config.json "${COLLECT_DIR}"/kubelet/config.json 2> /dev/null
       cp --force --recursive --dereference /etc/kubernetes/kubelet/config.json.d "${COLLECT_DIR}"/kubelet/config.json.d 2> /dev/null
+
+      cp --force --recursive --dereference /etc/kubernetes/kubelet/kubelet-config.json "${COLLECT_DIR}"/kubelet/config.json 2> /dev/null
```

Comment:
same destination file name as line 475?

Reply:
Should be safe to assume only one of those source paths exist, but this is for debug after all :) fixing...